packages/zowe-explorer/package.json

```diff
@@ -1710,6 +1710,12 @@
           "description": "%zowe.jobs.history%",
           "scope": "application"
         },
+        "zowe.jobs.confirmSubmission": {
+          "type": "boolean",
+          "default": true,
```

Comment:
🤔 Should the default be false? 
That way it's not a "breaking" change to the users

Reply:
Yes it should be `false`. Thanks for pointing that out, had it set to true for easy testing